### PR TITLE
Enable AWS Certificate Manager Support

### PR DESCRIPTION
**Pull request by sevenmachines**
This change adds in ACM support for bootstrap-cfn. This is usable
as an option when projects have access to the admin range of email
addresses on the domain or super-domain. It also includes docs and
a cloudformation stack for setting up an SES->S3->Lambda email
forwarding.

### CODE REVIEW

File: docs/ses_acm_validation.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+We can set up Amazon Simple Email Service (SES) to work together with ACM. 
+First we setup a route 53 MX record on the super-domain hosted zone to use the Simple Email Service (SES). 
+Add an SES ruleset that sends mail to an S3 bucket and triggers a Lambda function that will 
+forward the email from the bucket. This way the verification email can be sent on to real administrators.
+
+Using the SESLambdaForwarder cloudformation stack
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There’s a cloudformation template available for doing this,
```
Comment (mattpep) [inline docs/ses_acm_validation.md]: This is a &rsquo; - it should be &apos; (looks like a weird smart quote issue)

Comment (sevenmachines) [reply docs/ses_acm_validation.md]: updated

File: docs/ses_acm_validation.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+AWS Certification Manager (ACM) requires verification of the ownership of the domain name, 
+or alternatively, of the super-domain of the domain that you want the certificate for. 
+Currently the only way this is done is through email to the following addresses,
+
+* Whois domain, technical, and administrative contact
+* administrator, hostmaster, postmaster, webmaster, and admin at the requested domain or the super-domain if specified.
+
+http://docs.aws.amazon.com/acm/latest/userguide/gs-acm-validate. html
+
+![alt text](images/ses_acm_ses_sns.png "Certificate manager to simple email service pipeline")
```
Comment (yufangzhang) [inline docs/ses_acm_validation.md]: think image is in `images/acm_ses_sns.png`...

File: docs/ses_acm_validation.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With a lot of hosted zones we could end up doing extra work to add MX records to everyone 
+of them, luckily, we can use the super-domain instead of the actual domain to do verifications. 
+This means that instead of requiring setting up mail every hosted zone, we can simply set all 
+the sub-domains to verify on the super-domain. For example, mysite.mysubdomain.mydomain.com 
+sets validation domain to mydomain.com, so the verification emails will not be set to 
+admin@mysite.mysubdomain.superdomain.com, but instead to admin@superdomain.com.
+
+See http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/ aws-resource-certificatemanager-certificate.html
```
Comment (yufangzhang) [inline docs/ses_acm_validation.md]: a tiny space blocking the url....:p

File: docs/ses_acm_validation.md (hunk 2 above)
Comment (sevenmachines) [reply docs/ses_acm_validation.md]: think its acm_ses_sns.png :)

**Comment by sevenmachines** [comment]
@yufangzhang made the changes and fixed the docs. The image link should work once its in master

**Pull request merged**